components/esp_wifi_remote/idf_v5.5/Kconfig.slave_select.in

```diff
@@ -1,7 +1,10 @@
 # This file is auto-generated
     choice SLAVE_IDF_TARGET
     prompt "choose slave target"
+
+    default SLAVE_IDF_TARGET_ESP32C6 if IDF_TARGET_ESP32P4
```

Comment:
these files are generated.

I'll think about how we can inject these preferences.